automation/e2e-pipelines/example-pipelines-test.sh

```diff
@@ -11,16 +11,11 @@ EOF
 }
 
 function wait_for_pipelinerun() {
-  oc wait -n ${namespace} --for=condition=Succeeded=True pipelinerun -l pipelinerun="$1"-run --timeout=60m &
-  success_pid=$!
-
-  oc wait -n ${namespace} --for=condition=Succeeded=False pipelinerun -l pipelinerun="$1"-run --timeout=60m && exit 1 &
-  failure_pid=$!
-
-  if wait -n $success_pid $failure_pid ; then
+  oc wait -n ${namespace} --for=condition=Succeeded=True pipelinerun -l pipelinerun="$1"-run --timeout=60m
+  if ! $?; then
```

Comment:
Not sure if it improves readability, but you could put the command directly into the if condition.

Reply:
It will work for sure, but I would keep it as it is